runtime/runtime-params-estimator/estimate.sh

```diff
@@ -20,8 +20,13 @@ rm -rf /tmp/data
 
 set -ex
 
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd "${script_dir}"/../../
```

Comment:
I'm approving it but wondering why "${x}" and not "$x".

Reply:
If there's other var beigin with script_dir, say `s=foo`, `$script_dir` will become `foocript_dir`. Unlikely though.